src/Enums/UserRole.tsx

```diff
@@ -39,4 +39,19 @@ export const getUserRole = (roleFromCookie: string | undefined): UserRole => {
     default:
         return UserRole.STUDENT;
     }
+};
+
+export const getUserId = () => {
+    const userId = Cookies.get(CookieEnum.USERID);
+    let userIdValue = 0; 
+
+    if (userId === undefined || isNaN(userIdValue = parseInt(userId, 10))) {
```

Comment:
should we be using `_.isNil` instead of `=== undefined`
Is that just for isStrict? because `isNaN` should catch that